packages/terra-dropdown-button/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ChangeLog
 =========
+### Changed
+* Removed Hookshot to fix accessibility issues in dropdown menu.
```

Comment:
Can you move this under the unreleased section right below?

Reply:
Changed 447252c